### DISCUSSION TITLE
[Script request] Ubiquiti UNMS/UISP

### DISCUSSION THREAD

**Discussion by edwardusnigmus** [community-scripts/ProxmoxVE#620] [Request script] [answered]
### Application Name

UNMS/UISP

### Website

https://uisp.com/uisp-overview

### Description

UNMS is a free network management system for Ubiquiti devices with the exception of UniFi series, which has its own controller.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Whats the difference to https://community-scripts.github.io/ProxmoxVE/scripts?id=unifi ?

**Reply by Banjer** [reply]
The Unifi controller is only used for Unifi branded devices, not for the other devices they sell, and UNMS is more aimed at ISP's.

**Comment by Mason10198** [comment]
To add on to OP... UISP (previously "UNMS") is a self-hostable management application for UISP devices, similar to the Unifi Network Application for Unifi Network devices.

Ubiquiti has just recently terminated their free UISP hosting tier, and now requires a paid professional tier, or self-hosting in order to use the UISP management application. A large number of individuals who rely on UISP are now looking for easy self hosting options, so this would be perfect for Community-Scripts.

See [UISP - First Time Setup & Installation](https://help.ui.com/hc/en-us/articles/115012196527-UISP-First-Time-Setup-Installation) for more information.

**Comment by tremor021** [comment]
As far as i can see its just installing docker + bunch of containers:

![image](https://github.com/user-attachments/assets/b507e256-78f6-47a4-8b61-5e8f4df61a2e)

**Reply by enboig** [reply]
I tried installing manually but my it keeps restarting. I found a docker image able to run this (https://github.com/Nico640/docker-unms), but I couldn't make it run in a lxc.

**Comment by niekniek89** [comment]
Are there any plans yet to build/create an LXC script for UNMS?

**Reply by MickLesk** [reply, marked as answer]
no, its better to use it as docker, to many different containers needed
